### DISCUSSION TITLE
Is it possible to generate porosity in Neper?

### DISCUSSION THREAD

**Discussion by Geo572** [neperfepx/neper#706] [Q&A] [answered]
Hello @rquey, is it possible to generate grain-based models of different porosities (white circles) in Neper as you can see in picture?
Thank you.

<img width="602" alt="7C4DAFC1-B122-4e27-A028-2651DE09FB96" src="https://github.com/neperfepx/neper/assets/127503734/4af0ead2-4111-408b-8011-81381cade24e">

**Comment by rquey** [comment, marked as answer]
No, this has never been implemented.

**Reply by Geo572** [reply]
Thank you for your kind reply.
